masterlist.yaml

```
@@ -9547,6 +9547,56 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         udr: 24
 
+  - name: 'DWKaPoTunRace.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/40928/'
+        name: 'Races of Nirn - Kapo''tun SE'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/77612/'
+        name: 'Races of Nirn - Kapotun Reborn'
+    inc:
+      - name: 'Kapotun SOS.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0", >=)'
+      - name: 'Kapotun Vanilla Armor Patch.esp'
+        condition: 'version("DWKaPoTunRace.esp", "2.0", >=)'
+    msg:
+      - <<: *obsolete
+        subs: [ '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612/)' ]
+        condition: 'version("DWKaPoTunRace.esp", "2.0", <) and not checksum("DWKaPoTunRace.esp", 672EB2D2)'
+      - <<: *patchIncluded
+        subs:
+          - 'Schlongs of Skyrim'
+        condition: '(active("Schlongs of Skyrim - Core.esm") and version("DWKaPoTunRace.esp", "2.0", >=)) and not active("SOS - Kapotun Addon.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Touched by Dibella'
+          - '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612/)'
+        condition: '(active("TouchedbyDibella.esp") and checksum("DWKaPoTunRace.esp", 672EB2D2)) and not active("Kapotun TBD Tail.esp")'
```

These should be swapped now since `patch3rdParty` is first alphanumerically. Also, the extra parentheses are unnecessary due to how `and` is evaluated (i.e., they all have to be true for the entire expression to be true, grouping doesn't change anything), & single-entry lists should be written in flow style.

You could also swap the checksum function for `version("DWKaPoTunRace.esp", "2.0", <)`, or are you trying to avoid the message being displayed for older versions of your version?
```suggestion
      - <<: *patch3rdParty
        subs:
          - 'Touched by Dibella'
          - '[Races of Nirn - Kapotun Reborn](https://www.nexusmods.com/skyrimspecialedition/mods/77612/)'
        condition: 'active("TouchedbyDibella.esp") and checksum("DWKaPoTunRace.esp", 672EB2D2) and not active("Kapotun TBD Tail.esp")'
      - <<: *patchIncluded
        subs: [ 'Schlongs of Skyrim' ]
        condition: 'active("Schlongs of Skyrim - Core.esm") and version("DWKaPoTunRace.esp", "2.0", >=) and not active("SOS - Kapotun Addon.esp")'
```